firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -40,6 +41,7 @@ class AddressListViewModel: ObservableObject, FeatureFlaggable {
     @Published var isEditMode = false
 
     let windowUUID: WindowUUID
+    var webView: WKWebView?
```

Comment:
This sounds looks like a good solution, I tested locally and we can see an improvement.
However I would like to improve this since we usually don't like to keep UI elements in view model. I suggest having something like

```
import WebKit

protocol WebViewManaging {
    var webView: WKWebView { get }
    func loadLocalFile(_ filePath: String, relativeTo baseURL: URL)
    func resetWebView()
}
```
and a class that handle the preloading. Having the protocol makes it easier to mock the functionality. 
```
class WebViewManager: WebViewManaging {}
```

Reply:
Nit: could we call it CredentialWebView or AddressWebView 
Reason being this naming could confuse when searching webviews for the Tabs and credential has its own mechanism of injecting JS etc.